test/index-test.js

```diff
@@ -83,6 +87,18 @@ describe('transformPropsWith', function () {
     expect(node.textContent).toEqual('20')
   })
 
+  it('accepts __ref prop and remove it', function () {
```

Comment:
I still think that if we moved the transformation to a separate file and tested just it, as a pure function, it would be better